lib/errors.ts

```diff
@@ -125,7 +126,8 @@ export const GitErrorRegexes = {
     GitError.ProtectedBranchDeleteRejected,
   'error: GH006: Protected branch update failed for (.+).\nremote: error: Required status check "(.+)" is expected':
     GitError.ProtectedBranchRequiredStatus,
-  'error: GH007: Your push would publish a private email address.': GitError.PushWithPrivateEmail
+  'error: GH007: Your push would publish a private email address.': GitError.PushWithPrivateEmail,
+  'could not lock config file (.+): File exists': GitError.ConfigLockFileAlreadyExists
```

Comment:
It seems in _most_ cases above we've included the error type prefix as well. Could we do so here as well?

```suggestion
  'error: could not lock config file (.+): File exists': GitError.ConfigLockFileAlreadyExists
```